coverage/src/main/kotlin/org/rust/coverage/RsCoverageRunner.kt

```diff
@@ -46,6 +48,10 @@ class RsCoverageRunner : CoverageRunner() {
     companion object {
         private val LOG: Logger = Logger.getInstance(RsCoverageRunner::class.java)
 
+        @TestOnly
+        @Volatile
+        var lastProjectData: ProjectData? = null
```

Comment:
Looks like you don't need it at all and you may use `CoverageDataManager.getInstance(project).currentSuitesBundle.coverageData` instead